spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1415,7 +1415,18 @@ bom {
 			releaseNotes("https://github.com/pgjdbc/pgjdbc/releases/tag/REL{version}")
 		}
 	}
-	library("Prometheus Client", "0.16.0") {
+	library("Prometheus Client", "1.1.0") {
+		group("io.prometheus") {
+			modules = [
+					"prometheus-metrics-exposition-formats"
```

Comment:
I've requested the 1.x versions provide a BOM like the 0.x versions had. When that is available, this can be updated to manage the dependencies via the BOM.

Reply:
The BOM is available now since version 1.2.0. I've updated to use it in my last commit.